Backpack-SwiftUI/AppSearchModal/Classes/BPKAppSearchModal.swift

```diff
@@ -34,42 +35,42 @@ public struct BPKAppSearchModal: View {
         inputHint: String,
         results: BPKAppSearchModalResults,
         closeAccessibilityLabel: String,
+        inputPrefix: BPKSearchInputSummary.InputPrefix = .icon,
         onClose: @escaping () -> Void
     ) {
         self.title = title
         self._inputText = inputText
         self.inputHint = inputHint
         self.results = results
         self.closeAccessibilityLabel = closeAccessibilityLabel
+        self.inputPrefix = inputPrefix
         self.onClose = onClose
     }
     
     public var body: some View {
         VStack(spacing: .base) {
-            
             makeNavigationBar(title: title, closeAccessibilityLabel: closeAccessibilityLabel, onClose: onClose)
-            
             if results.showTextField {
-                BPKTextField(placeholder: inputHint, $inputText)
+                BPKSearchInputSummary(placeholder: inputHint, inputPrefix: inputPrefix, $inputText)
                     .inputState(textFieldState.inputState)
                     .accessibilityAddTraits(.isSearchField)
                     .focused($inputFieldIsFocussed)
                     .autocorrectionDisabled(true)
             }
-        
             switch results {
             case .loading(let loading):
                 AppSearchModalLoadingView(state: loading)
             case .content(let content):
-                AppSearchModalContentView(
-                    state: content,
-                    onScroll: onScroll(_:))
+                AppSearchModalContentView(state: content, onScroll: onScroll(_:))
                     .padding(.top, .md)
             case .error(let error):
                 AppSearchModalErrorView(state: error)
                     .padding(.horizontal, .md)
             }
         }
+        .onAppear {
+            inputFieldIsFocussed.toggle()
+        }
```

Comment:
I'm not sure we want this within the component, for example, hotels does not want this. it should be removed from here and leave it up to the clients defining this

Reply:
(if this is something you want to have)

Reply:
yeah, 100% should be removed 😅